Makefile

```diff
@@ -207,6 +207,10 @@ kind-tests: kind-delete-cluster kind-bootstrap-cluster-dev kind-deploy-controlle
 
 .PHONY: install-crds
 install-crds:
+	@echo installing olm
+	curl -L https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.26.0/install.sh -o $(LOCAL_BIN)/install.sh
+	chmod +x $(LOCAL_BIN)/install.sh
+	$(LOCAL_BIN)/install.sh v0.26.0
```

Comment:
Future TODO: it's not necessary, but it might be nice if this version were a variable